integration-test/src/python/local_test_runner/main.py

```diff
@@ -25,7 +25,7 @@
 ]
 # Retry variables in case the output is different from the input
 RETRY_COUNT = 5
```

Comment:
we can increase the RETRY_COUNT to allow the test has more chance while decreasing the RETRY_INTERVAL.